src/app/add-new-todo-list/add-new-todo-list.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-add-new-todo-list',
+  templateUrl: './add-new-todo-list.component.html',
+  styleUrls: ['./add-new-todo-list.component.scss'],
+})
+export class AddNewTodoListComponent {
+  List = [];
+  id = '';
+  listTitle = '';
+  item = [];
+
+  addList() {
+    this.List.push({
```

Comment:
Do not use push, it does not trigger change detection in angular.
What is the type of the object you are pushing?